nix/mobile/android/release.nix

```diff
@@ -52,7 +52,7 @@ in stdenv.mkDerivation rec {
       root = path;
       include = [
         "package.json" "yarn.lock" "metro.config.js"
-        "resources/.*" "translations/.*"
+        "resources/.*" "translations/.*" "src/js/.*" ".babelrc"
```

Comment:
No, it should not be necessary, since this is the Android Gradle build.
Are we sure this change shouldn't go to `nix/mobile/android/jsbundle/default.nix`?

Reply:
What file does it complain about exactly?

Reply:
So for using JavaScript Directives for creating [reanimated worklets](https://stackoverflow.com/questions/72500152/how-to-use-reanimated-worklet-directive-in-clojurescript/72513996#72513996) in the clojure project, I needed to use the js file.

For that, I created a new file `src/js/worklet-factory.js`. But when the android app is built it complains that this [file is not found](https://github.com/status-im/status-react/pull/13470#issuecomment-1151745897). 

This file is in the `"src/.*"` directory which we bundle in `nix/mobile/android/jsbundle/default.nix`, still it complains. Maybe because of how it is [imported](https://github.com/status-im/status-react/pull/13470#discussion_r907148059), this file acts more like a separate module.

Reply:
Then if only `src/js/worklet-factory.js` is necessary, why are you adding all of `src`?